tests/test_plotting.py

```diff
@@ -107,6 +107,62 @@ def make_triangle(t0, t1):
         self._compare_images(ax=ax, filename=filename)
 
 
+class TestPointPlotting(unittest.TestCase):
+
+    def setUp(self):
+
+        self.N = 10
+        self.points = GeoSeries(Point(i, i) for i in range(self.N))
+        values = np.arange(self.N)
+        self.df = GeoDataFrame({'geometry': self.points, 'values': values})
+
+    def test_default_colors(self):
+
+        ## without specifying values -> max 9 different colors
+
+        # GeoSeries
+        ax = self.points.plot()
+        cmap = get_cmap('Set1', 9)
+        expected_colors = cmap(range(9)*2)
```

Comment:
Hmm, I tested this with matplotlib on py3 and `cmap` happily takes an iterator instead of a list. But will make the change to be sure, and to retrigger travis